### PR TITLE
Fix syntax of migration guide after dark mode merge

### LINKED ISSUE #1782: Change of syntax in migration guide
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

It seems the syntax inside the migration guide was modified sice dark mode was merged.

Before dark mode:
```
## v5.2.3
### Components
- **Card**
- **Dropdowns**
```

After dark mode:
```
## v5.3.0
### Components
#### Alert
#### List group
```

### Reduced test cases

We will need to:
- Choose the syntax we want and/or need
- Fix parts where syntax is noth the chosen one

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

v5.2.3

### PR DESCRIPTION
**Pull request by MewenLeHo**
### Related issues

Closes #1782 

### Description

Fix change of syntax for subtitles.

### Motivation & Context

Keep the same syntax everywhere.

### Types of change

- Bug fix (non-breaking which fixes an issues)

### Live previews

- [https://deploy-preview-1784--boosted.netlify.app/docs/5.2/migration/](https://deploy-preview-1784--boosted.netlify.app/docs/5.2/migration/)

#### Contribution

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

#### Accessibility

- [x] My change follows accessibility good practices; I have at least run axe

#### Design

- (NA) My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
- (NA) My change is compatible with responsive display

#### Development

- (NA) My change follows the [developer guide](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Developer-guide)
- (NA) I have added JavaScript unit tests to cover my changes

#### Documentation

- [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [x] My change introduces changes to the migration guide
- (NA) My new component is added in Storybook
- (NA) My new component is compatible with RTL
- (NA) Manually run [BrowserStack tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/browserstack.yml)
- (NA) Manually test browser compatibility with BrowserStack (Chrome >= 60, Firefox >= 60 (+ ESR), Edge, Safari >= 12, iOS Safari, Chrome & Firefox on Android)
- [x] Code review
- (NA) Design review
- (NA) A11y review

#### After the merge

- [ ] Manually launch [Percy tests](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/workflows/percy.yml)

### CODE REVIEW

**Comment by MewenLeHo** [comment]
It seems that the syntax was also different in the `v5.1.0` part too. Should we fix it too?

**Comment by hannahiss** [comment]
**In v5.1.0, we have:**
v5.1.0 => h2
Components => h2
Component1 => h3

**since v5.20:** 
v5.2.0 => h2
Components => h3
Component1 => list element (bold)

**Since v5.3.0:** 
v5.3.0 => h2
Components => h3
Component1 => h4

I have a preference for the latest version, and I think it should be the same organization for all versions?

**Comment by louismaximepiton** [comment]
For me there isn't any need for v5.1.0 and before since we don't want people to migrate to this version + we changed the way we build this migration guide. 

For the semantical hierarchy, I personnaly don't know what is best here. Even if we should use titles, if we keep all items as titles, the table of content will be a book and might be a bit long to go through by tabbing. Furthermore for people with AT, since we have an issue to add some different labels on those links, it might be awful to understand.

So I vote neutral 😅

**Comment by MewenLeHo** [comment]
I also think we can keep the fix in this PR to keep it consistant with what was done since the new version of the guide.

File: site/content/docs/5.2/migration.md [hunk 1]
```diff
@@ -46,14 +46,12 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
 ### Components
 
-#### Alert
-
+- **Alert**
 - Alert variants are now styled via CSS variables.
 
 - <span class="badge text-warning-emphasis bg-warning">Deprecated</span> The `.alert-variant()` mixin is now deprecated. We now [use the Sass loop]({{< docsref "/components/alerts#sass-loop" >}}) directly to modify the component's default CSS variables for each variant.
```
Comment (louismaximepiton) [inline site/content/docs/5.2/migration.md:52]: Missing some tabulations to be fine, same below

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Small changes to apply.

We should talk about the title in #1519 maybe ? And try to tackle this before 5.3.0

**Review by louismaximepiton — APPROVED** [review]
LGTM !

**Comment by MewenLeHo** [comment]
Should we also change the 'Deprecated' badge to 'Breaking'? I am not sure if a depreacted CSS class or mixin is a breaking as a deprecated Sass variable?

screenshot-localhost_9001-2023 02 03-11_15_42

**Comment by julien-deramond** [comment]
Is this PR ready for final review?

**Comment by MewenLeHo** [comment]
> Is this PR ready for final review?

Unfortunately no, I had to add some modifications after @louismaximepiton last review.
There is also a remaining question in the last comment, you may have an answer for it?

**Comment by julien-deramond** [comment]
> There is also a remaining question in the last comment, you may have an answer for it?

Kept it because it was used by Bootstrap but you can change it to "Breaking" I don't mind. This might be breaking or not depending in the users. In this case, most of the time, it won't have any impact for anyone.

File: site/content/docs/5.2/migration.md [hunk 2]
```diff
@@ -134,14 +132,14 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
   </details>
 
 - <details class="mb-2">
-    <summary><span class="badge bg-danger">Breaking</span> Deprecated CSS variables:</summary>
+    <summary><span class="badge text-danger-emphasis bg-danger">Breaking</span> Deprecated CSS variables:</summary>
```
Comment (louismaximepiton) [inline site/content/docs/5.2/migration.md:135]: Should we introduce this `.text-danger-emphasis` on all the badges ? Might be done on Bootstrap side first IMO.

Comment (julien-deramond) [reply site/content/docs/5.2/migration.md:135]: Tried it in https://github.com/twbs/bootstrap/pull/38013. In the meantime, let's keep these badges close to Bootstrap in this PR.

**Review by louismaximepiton — APPROVED** [review]

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**